ansible/library/dut_basic_facts.py

```diff
@@ -44,6 +44,8 @@ def main():
         results['is_multi_asic'] = device_info.is_multi_npu()
         results['num_asic'] = device_info.get_num_npus()
         results.update(device_info.get_sonic_version_info())
+        results['kernel_version'] = results['kernel_version'].split('-')[0]
+        results['is_supervisor'] = device_info.is_supervisor()
```

Comment:
Since new variables are added, please update the conditional mark document to ensure that it is up to date.

Reply:
done